### DISCUSSION TITLE
What's the rationale for forcing the user to define the cheat codes interface explicitly?

### DISCUSSION THREAD

**Discussion by jamierumbelow** [foundry-rs/foundry#1379] [Q&A] [answered]
Hi there!

Thanks for the great work.

I'm finding the 'copy-and-paste signatures into an `interface` I define' approach a little unergonomic.

What's the rationale for this decision? Avoiding namespace clash with `Vm` or `CheatCodes` or something?

Or is there a `cheat-codes` import I'm missing that gives you a complete `Vm` object to work with?

**Comment by cruzdanilo** [comment, marked as answer]
```solidity
import { Vm } from "forge-std/Vm.sol";
```

https://github.com/foundry-rs/forge-std

**Reply by jamierumbelow** [reply]
Fantastic, thank you!
